ddocs/medic-gateway-wt/_id

```diff
@@ -0,0 +1 @@
+_design/medic-gateway-wt
```

Comment:
Should I know what `wt` stands for? Also, what's the specific reasoning behind this being in its own ddoc? Because it doesn't need to run unless it's an SMS project? If so we might want to name the ddoc something that makes that clear.

Reply:
Any suggestions?  `webapp-terminating-sms-by-id`?

Reply:
If it's specifically for SMS projects maybe just `medic-sms` or `medic-sms-views` or something?

Reply:
It's specifically a list of all the gateway IDs (`doc.sms_message.gateway_ref`) of sms messages that were sent by 3rd parties _to_ webapp via the gateway.  Maybe I should just throw these into another view, e.g. `doc_by_type`, and invent a pseudo-type for them (e.g. `webapp-terminating-sms-message`.)

Reply:
Right, the point I'm trying to make is that when we name the ddoc (I'm talking about the ddoc name, not the view name) we should take into account _why its a separate ddoc_ compared to the rest of our application, and reflect that in the name. So `webapp-terminated-sms-by-id` sounds like a potential view name, which is a different naming conversation entirely.